assets/upgradePatches.json

```diff
@@ -40,6 +40,21 @@
       "jsonPatchApplyOptions": {
         "allowMissingPathOnRemove": true
       }
+    },
+    {
+      "semverRange": "<1.14.0",
+      "jsonPatch": [
+        {
+          "op": "test",
+          "path": "/spec/featureGates/deployVmConsoleProxy",
+          "value": false
+        },
+        {
+          "op": "add",
+          "path": "/spec/enableTokenGenerationApi",
+          "value": false
+        }
+      ]
```

Comment:
I wanted to do this update with minimal surprise for the user. 
The problem is, as you've written, that we don't know if user decided to disable the feature, or just left it default.
If it was intentionally disabled, it would be a surprise to enable it on update.

But maybe the risk is low, and we can enable it.